Header header
float32[] A
float32[] B
float32[] C
float32[] K
float32[] L
float32[] Z
float32 ref_points
float32 Ts


# This is based on a trajectory that is transformed from a certain frame to the rover frame at time 0 in
# the following sense: x, y, psi start positions are 0,0,0.
# x positive is strait ahead, y positive is to the right, psi starts positive clockwise from x axis.



